pkgs/games/anki/bin.nix

```diff
@@ -49,28 +49,39 @@ let
   };
 
   passthru = { inherit sources; };
-in
 
-if stdenv.isLinux then buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
-  name = "anki";
+  fhsUserEnvAnki = buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
+    name = "anki";
 
-  # Dependencies of anki
-  targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
+    # Dependencies of anki
+    targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
 
-  runScript = writeShellScript "anki-wrapper.sh" ''
-    exec ${unpacked}/bin/anki
-  '';
+    runScript = writeShellScript "anki-wrapper.sh" ''
+      exec ${unpacked}/bin/anki
+    '';
 
-  extraInstallCommands = ''
-    mkdir -p $out/share
-    cp -R ${unpacked}/share/applications \
-      ${unpacked}/share/man \
-      ${unpacked}/share/pixmaps \
-      $out/share/
-  '';
+    extraInstallCommands = ''
+      mkdir -p $out/share
+      cp -R ${unpacked}/share/applications \
+        ${unpacked}/share/man \
+        ${unpacked}/share/pixmaps \
+        $out/share/
+    '';
+
+    inherit meta passthru;
+  });
+
+  fhsUserEnvAnkiWithVersion = fhsUserEnvAnki.overrideAttrs (oldAttrs: {
+    # buildFHSUserEnv doesn't have an easy way to set the version of the
+    # resulting derivation, so we manually override it here.  This makes
+    # it clear to end users the version of anki-bin.  Without this, users
+    # might assume anki-bin is an old version of Anki.
+    name = "${pname}-${version}";
```

Comment:
This changes the resulting name of the derivation from `anki` to `anki-bin-2.1.60`.

I don't have a strong opinion whether the derivation should be named `anki-2.1.60` or `anki-bin-2.1.60`.  I'd be fine with either.  (But I do feel strongly that the version should be included.)